vegafusion-python/tests/test_plan_resolver.py

```
@@ -0,0 +1,710 @@
+from vegafusion import ExternalDataset, PlanResolver
+import pytest
+from inline_snapshot import snapshot
+
+from vegafusion.plan_resolver import (
+    ResolvedPlan,
+    inline_table_scan_node,
+    unparse_to_sql,
+)
+
+
+def setup_module(module: Any) -> None:
```

Ruff (0.15.6)

[warning] Unused function argument: `module`

(ARG001)